src/util/Gatdoc.jl

```diff
@@ -0,0 +1,31 @@
+module Gatdoc 
+
+using Markdown
+
+export @gatdoc, gatdoc_impl 
+
+function gatdoc_impl(theory)
+ 
+  vec = split(repr(theory), "\n")
```

Comment:
You should overload `repr` for theories so that you don't have to do this string munging jankery.